changes/probe.py

```diff
@@ -58,11 +68,20 @@ def has_changelog():
 
 
 def has_readme():
-    """README.md"""
+    """README"""
+    readme = None
+    message = 'Create a README file'
+    for filename in os.listdir("."):
+        readme = re.match(r"README($|\.\w{2,})", filename, re.I)
```

Comment:
Any reason you're not using `glob` here, like you do in the [test](https://github.com/michaeljoseph/changes/pull/106/files#diff-b5842c8912eae6fcc58855f0e382242cR45)?